### PR TITLE
feat(consonant-blocks): article blocks added

### PR DESCRIPTION
**Pull request by So1ene**
- XD Spec: https://xd.adobe.com/view/7bed0802-e0e1-4b31-a348-01fd945980db-2ebe/screen/1df30115-92b1-48c8-8698-001a99d7d4dc/specs/

- Test URL: https://article-blocks--pages--webistry-development.hlx3.page/stock/en/artisthub/drafts/articles/article-example

### Article Blocks Added: ###
- Title
image
- Images 
image
- Textbody
- Quote (with a Gray variant)
- List
image
- Share
- Tags
image

### CODE REVIEW

File: templates/consonant/blocks/images/images.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    margin-right: auto;
+    display: inline-block;
+}
+
+main .images-container > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
```
[the remark this answers is not in the document]
Comment (seanchoi-workfront) [reply templates/consonant/blocks/images/images.css:13]: Specially under the cards-horizontal, it looks off the desgin.

Your result:
image

XD:
image

**Review by seanchoi-workfront — CHANGES_REQUESTED** [review]
Mostly looking great, I found few things aren't matching with XD. Please take a look.

**Comment by So1ene** [comment]
> Mostly looking great, I found few things aren't matching with XD. Please take a look.

Thank you for your feedback, I made the fixes, please look again when you get the chance :) @seanchoi-workfront

**Review by seanchoi-workfront — APPROVED** [review]
Looks good

**Pull request merged**